go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/metalmatze/slo-libsonnet
```

Comment:
Do we ned this? I don't think so

Reply:
It failed to initialize without the root go.mod file at least. We might be able to remove it now, but I had to have it for the first bingo deps.